runtime/Jsx.res

```diff
@@ -38,3 +38,617 @@ type component<'props> = componentLike<'props, element>
 
 /* this function exists to prepare for making `component` abstract */
 external component: componentLike<'props, element> => component<'props> = "%identity"
+
+module DOMStyle = JsxDOMStyle
+module Event = JsxEvent
+module DOM = {
+  type style = DOMStyle.t
+  type domRef
```

Comment:
I think it is problematic that the types defined here (other than `style`) are different from those in `JsxDOM`. They should all be aliases.